ui/src/components/time/QTime.js

```diff
@@ -379,9 +379,11 @@ export default Vue.extend({
     __goToNextView () {
       if (this.view === 'Hour') {
         this.view = 'Minute'
+        this.__setMinute(0)
```

Comment:
Before setting minute or second to 0 (zero), I think you should verify they are null, in which case it would be fine to do those actions. 
What if a User set a time that had minutes or seconds to the model? Then these would be lost.